typescript/ecs/fargate-service-with-local-image/index.ts

```diff
@@ -0,0 +1,28 @@
+import ec2 = require('@aws-cdk/aws-ec2');
+import ecs = require('@aws-cdk/aws-ecs');
+import cdk = require('@aws-cdk/cdk');
+import path = require('path');
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'FargateServiceWithLocalImage');
+
+// Create VPC and Fargate Cluster
+// NOTE: Limit AZs to avoid reaching resource quotas
+const vpc = new ec2.VpcNetwork(stack, 'MyVpc', { maxAZs: 2 });
+const cluster = new ecs.Cluster(stack, 'Cluster', { vpc });
+
+// Instantiate Fargate Service with a cluster and a local image that gets 
+// uploaded to an S3 staging bucket prior to being uploaded to ECR.
+// A new repository is created in ECR and the Fargate service is created
+// with the image from ECR.
+const fargateService = new ecs.LoadBalancedFargateService(stack, "FargateService", {
+  cluster,
+  image: ecs.ContainerImage.fromAsset(stack, "local-image" , { 
+    directory: path.join(__dirname, 'local-image') 
+  })
+});
+
+// Output the DNS where you can access your service
+new cdk.CfnOutput(stack, 'LoadBalancerDNS', { value: fargateService.loadBalancer.dnsName });
```

Comment:
Nit: You actually don't need this line, since it exists in the base load balanced service construct: https://github.com/awslabs/aws-cdk/blob/master/packages/%40aws-cdk/aws-ecs/lib/load-balanced-service-base.ts#L124

Otherwise, when you deploy you get this:
```
Outputs:
FargateServiceWithLocalImage.FargateServiceLoadBalancerDNS9433D5F6 = Farga-Farga-T1HBD97D0CHZ-xxxxx308.us-west-2.elb.amazonaws.com
FargateServiceWithLocalImage.LoadBalancerDNS = Farga-Farga-T1HBD97D0CHZ-xxxxx308.us-west-2.elb.amazonaws.com
```

Reply:
Done.